### PR TITLE
fix: actually run all unit tests in CI

### PR DESCRIPTION
**Pull request by DaniPopes**
oops

### CODE REVIEW

**Review by mattsse — APPROVED** [review]

File: crates/cast/bin/cmd/call.rs [hunk 1]
```diff
@@ -254,11 +254,11 @@ mod tests {
     #[test]
     fn can_parse_call_data() {
         let data = hex::encode("hello");
-        let args: CallArgs =
-            CallArgs::parse_from(["foundry-cli", "--data", format!("0x{data}").as_str()]);
-        assert_eq!(args.data, Some(data.clone()));
+        let args = CallArgs::parse_from(["foundry-cli", "--data", data.as_str()]);
+        assert_eq!(args.data, Some(data));
```
Comment (DaniPopes) [inline crates/cast/bin/cmd/call.rs:258]: We don't strip prefixes by default anymore

File: crates/forge/bin/cmd/install.rs [hunk 2]
```diff
@@ -505,6 +505,7 @@ mod tests {
     use tempfile::tempdir;
 
     #[test]
+    #[ignore = "slow"]
```
Comment (DaniPopes) [inline crates/forge/bin/cmd/install.rs:508]: I don't really want these tests in unit tests, same for some that call providers or fork.

File: crates/forge/bin/cmd/script/mod.rs [hunk 3]
```diff
@@ -703,21 +703,14 @@ mod tests {
...
-            "--sig",
-            "0x522bb704000000000000000000000000f39fd6e51aad88f6f4ce6ab8827279cfFFb92266",
-        ]);
-        assert_eq!(
-            args.sig,
-            "522bb704000000000000000000000000f39fd6e51aad88f6f4ce6ab8827279cfFFb92266"
-        );
+        let sig = "0x522bb704000000000000000000000000f39fd6e51aad88f6f4ce6ab8827279cfFFb92266";
+        let args = ScriptArgs::parse_from(["foundry-cli", "Contract.sol", "--sig", sig]);
+        assert_eq!(args.sig, sig);
```
Comment (DaniPopes) [inline crates/forge/bin/cmd/script/mod.rs:708]: Prefix again

File: crates/forge/bin/cmd/verify/etherscan/mod.rs [hunk 4]
```diff
@@ -512,7 +512,7 @@ mod tests {
                 &config,
             )
             .unwrap();
-        assert_eq!(client.etherscan_api_url().as_str(), "https://api-testnet.polygonscan.com/?/");
+        assert_eq!(client.etherscan_api_url().as_str(), "https://api-testnet.polygonscan.com/");
```
Comment (DaniPopes) [inline crates/forge/bin/cmd/verify/etherscan/mod.rs:515]: Don't know what this is about

Comment (mattsse) [reply crates/forge/bin/cmd/verify/etherscan/mod.rs:515]: this is outdated/no longer required, there were some cloudflare bugs with trailing /

File: crates/cast/bin/cmd/call.rs (hunk 1 above)
Comment (mattsse) [reply crates/cast/bin/cmd/call.rs:258]: I think we should?

Comment (mattsse) [reply crates/cast/bin/cmd/call.rs:258]: ah will be taken care of by hex decoding anyway so is fine

**Pull request merged**